DHIS2/cloner/dhis2_clone

```diff
@@ -80,8 +85,10 @@ def get_args():
     add('--no-backups', action='store_true', help="don't make backups")
     add('--no-webapps', action='store_true', help="don't clone the webapps")
     add('--no-db', action='store_true', help="don't clone the database")
+    add('--no-postprocess', action='store_true', help='don\'t execute postprocessing actions')
     add('--manual-restart', action='store_true', help="don't stop/start tomcat")
     add('--post-sql', nargs='+', default=[], help='sql files to run post-clone')
+    add('--post-clone-scripts', action='store_true', help='execute all py and sh scripts under post_clone_scripts_dir folder')
```

Comment:
It actually runs only the scripts in that directory, not in subdirectories, so maybe instead of "all [...] under" you could say "all [...] in the post-clone-scripts-dir folder".

Also, where is the `--post-clone-scripts-dir` argument added? I can't find it.

Reply:
as per our previous conversation, --post-clone-scripts-dir is not treated as an argument, to follow what the rest of the dirs are doing. Accepting the re-wording suggestion